### ISSUE TITLE
App Crash with LibreWolf

### ISSUE THREAD

**Issue by sinnx3** [Chatterino/chatterino2#4108]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

I'm having trouble keeping Chatterino [base] and Chatterino [7tv] open when loading up twitch channels on LibreWolf, Chatterino appears to be crashing (without a crash report) and fails to relaunch when relevant relaunch setting is set in the configuration.

I have tried reinstallation of both Chatterino versions alongside LibreWolf and to no avail have I found a fix.

### Screenshots

Uploading 2022-11-04 02-05-49.mp4…

### OS and Chatterino Version

Chatterino 7.3.5 (commit 132b406cc), LibreWolfe Build 106.4-1, Windows 10 Home Version 22H2 Build 19045.2130

**Comment by Felanbird** [comment]
Does this issue persist with the Browser Extension disabled?

**Comment by sinnx3** [comment]
> Does this issue persist with the Browser Extension disabled?

No, issue does not persist when Chatterino Browser Extension is disabled.

**Comment by Felanbird** [comment]
> > Does this issue persist with the Browser Extension disabled?
> 
> No, issue does not persist when Chatterino Browser Extension is disabled.

The Browser Extension has been more-or-less abandoned since 2019, so I'm not too surprised you are running into issues on something that isn't native Chrome or Firefox.
